### ISSUE TITLE
`analyze_colvars` gives a warning when `extra_args` are used

### ISSUE THREAD

**Issue by danielinteractive** [Roche/rtables#76]
`analyze_colvars` gives a warning when `extra_args` are used in the `build_table` step. 

When I tried to debug this, then I found that the `exargs` list only has 5 elements instead of the required 6 in the example below, when passing to the `mapply` call at the end of 
`gen_rowvalues(dfpart, datcol = datcol, cinfo, func, splextra = splextra, takesdf = takesdf, baselines, inclNAs = inclNAs)`. 

Example:

```r
library(rtables)

dta_test <- data.frame(
  USUBJID = rep(1:6, each = 3),
  PARAMCD = rep("lab", 6*3),
  AVISIT = rep(paste0("V", 1:3), 6),
  ARM = rep(LETTERS[1:3], rep(6, 3)),
  AVAL = c(9:1, rep(NA, 9)),
  CHG = c(1:9, rep(NA, 9))
)

## Default output within a `rtables` pipeline.
l <- basic_table() %>%
  split_cols_by("ARM") %>%
  split_rows_by("AVISIT") %>%
  split_cols_by_multivar(vars = c("AVAL", "CHG")) %>%
  analyze_colvars(mean, extra_args = list(na.rm = TRUE))

build_table(l, dta_test)
```

Session info:
```r
R version 3.6.1 (2019-07-05)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Debian GNU/Linux 9 (stretch)

Matrix products: default
BLAS/LAPACK: /usr/lib/libopenblasp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=C              LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] rtables_0.3.2.17.9014 magrittr_1.5         

loaded via a namespace (and not attached):
[1] compiler_3.6.1  htmltools_0.4.0 tools_3.6.1     Rcpp_1.0.3      digest_0.6.23   packrat_0.5.0   rlang_0.4.3     purrr_0.3.3 
```

**Comment by danielinteractive** [comment]
Note that this leads to errors when the `extra_args` controls the results returned by `afun`, because then the number of results can be different between the last column and the first 5 in the example above, I suppose.

**Comment by gmbecker** [comment]
This should be fixed in 0.3.2.17.9014 and there are a wider range of tests around the various ways extra args can be specified.

If you need to pass different arguments to different functions, say a to funa and b to funb, you'll need to do `extra_args = list(list(a = 5), list(b=10))`, but if you want your extra args to be passed identically to every analysis function, you can do `extra_args = list(na.rm = TRUE)` as in your example, and it works. There is no way to mix arguments that should go to all functions and arguments that should only go to some, and there isn't likely to be such a way.
